docs/src/main/asciidoc/security-oidc-auth0-tutorial.adoc

```diff
@@ -0,0 +1,587 @@
+[id="security-oidc-auth0-tutorial"]
+= Protect Quarkus web application by using Auth0 OpenID Connect provider
```

Comment:
Missing articles in title.
```suggestion
= Protect a Quarkus web application by using the Auth0 OpenID Connect provider
```

Reply:
@michelle-purcell Hi Michelle, this suggestion won't work, this is actually referring to an OIDC provider named `Auth0`